portfolio/src/main/java/com/google/sps/servlets/user.java

```diff
@@ -0,0 +1,25 @@
+package com.google.sps.servlets;
+
+public final class user {
```

Comment:
`User` - class names should be upper-cased. However, `Contact` or `Request` would be more appropriate names. This class doesn't represent a user, it represents individual message posts.

Also, I don't see this class used anywhere? Maybe it was intended for converting Datastore data to JSON?